### PR TITLE
Improvements to README.md, docker run command, and Dockerfile

### PR DESCRIPTION
**Pull request by wesfloyd**

### CODE REVIEW

File: segmentation_testbed/test1.txt
Comment (jsolly) [inline segmentation_testbed/test1.txt:1]: I think this was just a test. Suggest removing from the PR.

File: segmentation_testbed/README.md [hunk 1]
```diff
@@ -50,24 +50,36 @@ Going off the example above, you would run the following command:
...
-$ docker run --rm -it -v ../../data/landsatSceneMosaics/LC08_L1TP_001028_20220615_20220627_02_T1_mosaic.tif:/project/inputs segmentation_testbed /bin/bash
-$ docker run --rm -v ../../data/landsatSceneMosaics/LC08_L1TP_001028_20220615_20220627_02_T1_mosaic.tif:/project/inputs segmentation_testbed
+```
+
+Execute the docker container to test
+```shell
+#Interactive Mode
+docker run --rm -it -v $PWD/../data/LC08_L1TP_001028_20220615_20220627_02_T1_mosaic.tif:/project/inputs/LC08_L1TP_001028_20220615_20220627_02_T1_mosaic.tif segmentation_testbed /bin/bash
+#Non-interactive Mode
+docker run --rm -v $PWD/../data/LC08_L1TP_001028_20220615_20220627_02_T1_mosaic.tif:/project/inputs/LC08_L1TP_001028_20220615_20220627_02_T1_mosaic.tif segmentation_testbed
```
Comment (jsolly) [inline segmentation_testbed/README.md:98]: Let's just assume they have the file available without a complicated directory structure.
# Interactive Mode 
docker run --rm -it -v $PWD/LC08_L1TP_001028_20220615_20220627_02_T1_mosaic.tif:/project/inputs/LC08_L1TP_001028_20220615_20220627_02_T1_mosaic.tif segmentation_testbed /bin/bash

# Non-interactive Mode
docker run --rm -v $PWD/LC08_L1TP_001028_20220615_20220627_02_T1_mosaic.tif:/project/inputs/LC08_L1TP_001028_20220615_20220627_02_T1_mosaic.tif segmentation_testbed

File: segmentation_testbed/test1.txt
Comment (wesfloyd) [reply segmentation_testbed/test1.txt:1]: agreed, good catch

File: segmentation_testbed/README.md (hunk 1 above)
Comment (wesfloyd) [reply segmentation_testbed/README.md:98]: kk - can you say more on what you mean by not having a complicated directory structure?

**Review by jsolly — APPROVED** [review]

File: segmentation_testbed/README.md (hunk 1 above)
Comment (jsolly) [reply segmentation_testbed/README.md:98]: In my example, I had the imagery mosaic in a folder 2 directories higher than where the command was being executed. If someone is following along, they can simply pass the mosaic.tif in without needing to do something like `../../data`

**Comment by jsolly** [comment]
Thanks for this PR!

**Pull request merged**